ansible/roles/riakcs/configure/templates/riakcs.conf.j2

```diff
@@ -421,7 +421,7 @@ platform_log_dir = /var/log/riak-cs
 ## 
 ## Acceptable values:
 ##   - text
-nodename = riakcs-{{ inventory_hostname_short }}@{{ inventory_hostname }}
+nodename = riakcs@{{ inventory_hostname }}
```

Comment:
this change may need to be made conditional to accommodate envs that are using the old naming style (some of those envs are also using bitcask, so that needs to be addressed before this PR can be merged as well).